src/containers/ProfilePage/ProfilePage.js

```diff
@@ -252,7 +254,7 @@ export const MainContent = props => {
           </ul>
         </div>
       ) : null}
-      {isMobileLayout ? (
+      {hideReviews ? null : isMobileLayout ? (
```

Comment:
I'm assuming here that if the user doesn't have viewing rights i.e. can't see reviews, we want to hide the whole review component instead of showing the review headings with 0 reviews. Let me know if that assumption is correct, or if we do want to show the review headings here either way!

<img width="675" alt="Screenshot 2024-10-01 at 12 13 27" src="https://github.com/user-attachments/assets/8735a1fc-a49a-44c8-b2f3-d2801fd98961">

Reply:
Makes sense to me 👍